examples/sample-app/cleanup.sh

```diff
@@ -1,10 +1,15 @@
 #!/bin/sh
 
 echo "Killing openshift all-in-one server ..."
-pkill -x openshift
+sudo pkill -x openshift
+
+echo "Stopping all k8s docker containers on host ..."
+sudo docker ps --format='{{.Names}}' | awk 'index($NF,"k8s_")==1 { print $1 }' | xargs -l -r sudo docker stop
```

Comment:
i suggested it because many people run "sudo openshift start", so either you need to mandate they also run "sudo cleanup.sh" or cleanup.sh itself needs to assert sudo everywhere.  I think that makes more sense than assuming users are members of the docker group.